src/main/bash/bliman-launch-helpers.sh

```diff
@@ -47,6 +47,7 @@ function __bliman_set_env_repo() {
 	git clone "https://github.com/$BLIMAN_NAMESPACE/BeSLab" /tmp/BeSLab
 	cp /tmp/BeSLab/beslab/0.0.1/* "$HOME/.besman/envs/"
 	source "$HOME/.besman/bin/besman-init.sh"
+        bes list
```

Comment:
Better to hide the output.

Reply:
I think its better to keep output as it will make as confirm which mode is being used. We will hide later on if it sees annoying.